dxfuse.go

```diff
@@ -1415,6 +1411,10 @@ func (fsys *Filesys) readRemoteFile(ctx context.Context, op *fuseops.ReadFileOp,
 	// See if the data has already been prefetched.
 	// This call will wait, if a prefetch IO is in progress.
 	len := fsys.pgs.CacheLookup(fh.hid, op.Offset, endOfs, op.Dst)
+	// log received length if less than requested
+	if fsys.options.Verbose && int64(len) < reqSize {
```

Comment:
I have concerns about this line, Go treats nil as a non-false value. 
Not sure if it's still the case, just wanted to call it out

Reply:
shouldn't this be fine as `Verbose` is `bool` which should default to `false`?

Reply:
You're right, that makes sense 👍